package.json

```diff
@@ -30,6 +30,8 @@
     "@docusaurus/utils-validation": "^2.0.0-beta.4",
     "@easyops-cn/autocomplete.js": "^0.38.1",
     "@node-rs/jieba": "^1.6.0",
+    "@mdi/js": "^6.6.96",
```

Comment:
Not sure if you want this dep, it might be overkill for just one icon.

Reply:
Um, yes, I think it's overkill. Can we just use a character like `✕` (`&#x2715;`) or `✖` (`&#10006;`)?

Reply:
I just realized that we can just use `<input type="search">` to enable a native cancel button. This can also fix the autocomplete value change issue.

On the other hand, we have to fix a problem of cancel button been overlapped by the search hints `<kbd>`. We can hide the hints when the input has value.

Reply:
Ah sorry I missed the second comment, I didn't think the characters looked very good so I just embedded the svg but maybe that is also dirty. I will look at the search input type, but I think maybe it doesn't have a cancel button on all browsers? Will fix the shortcut, I forgot about it because I had hidden it 😅